mars-mission/src/Components/forms.js

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import { countriesList } from "./countriesList/countries";
+import { diets } from "./Dietary/diets";
+import "./css/forms.css";
+
+class Form extends React.Component {
+  state = {
+    fullName: "",
+    date: {},
+    countries: {},
+    diet: {},
+    bodyResponse: "",
+    formCompleted: false
+  };
+
+  handleChange = e => {
+    // debugger;
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  };
+
+  populateDiets = () => {
+    return diets.map(diet => {
+      return <option value={diet}>{diet}</option>;
+    });
+  };
+
+  populateCountries = () => {
+    return countriesList.map(countries => {
+      return <option value={countries.code}>{countries.name}</option>;
+    });
+  };
```

Comment:
These maps are probably giving a warning about needing a key. When you do map you'll want to add a uniq key property to whatever is being made. In this case you would add key={countires.code} as well into your option. Looks good though.